index.html

```diff
@@ -95,7 +95,9 @@
         <div class="nav-login">
           <a
             class="btn-login"
-            href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+            id="user-login"
+            onclick="checkWindowURL()"
```

Comment:
Let's not wait until the user clicks on the button. We want the link to be there, whether the user click it or not.

Reply:
Will do these changes